packages/marketplace/src/resources/snippets.ts

```diff
@@ -77,4 +77,10 @@ export default [
     "code": ".main-rootlist-rootlistItemOverlay {\n  display: none;\n}",
     "preview": "resources/assets/snippets/fix-playlist-hover.png",
   },
+  {
+    "title": "Disable Homepage Recommendation",
+    "description": "Disable Homepage Recommendation charts",
+    "code": "section[data-testid='home-page'] .main-shelf-shelf:not([aria-label='Recently played'], [aria-label='Your playlists']) { display: none !important; }",
+    "preview": "resources/assets/snippets/disable_recommendation.png",
```

Comment:
Use `kebab-case` for file names for uniformity, just a suggestion.

Reply:
Make the preview image background transparent and delete the cluster also.
![image](https://user-images.githubusercontent.com/77577746/180603545-86596682-42e7-47bf-a111-dbc1cc6289cb.png)